src/seml/experiment/config.py

```diff
@@ -800,10 +816,16 @@ def read_config(config_path: Union[str, Path]):
     if len(slurm_list) == 0:
         slurm_list.append({})
 
-    return seml_dict, slurm_list, config_dict
+    return (
```

Comment:
I really dislike casts, can we cast once into a type that resembles the config from yaml?